README.md

```diff
@@ -95,6 +95,7 @@ dependencies {
 ### Compatibility Matrix
  Appium Java Client        | Selenium client
 ---------------------------|-----------------
+                           | `4.19.0`
```

Comment:
9.3.0?

Reply:
need to check, probably `9.2.1`, but let's add the version with release to avoid confusion